pydm/widgets/qtplugin_base.py

```diff
@@ -157,7 +159,9 @@ def icon(self):
         """
         Return a QIcon to represent this widget in Qt Designer.
         """
-        return QtGui.QIcon()
+        if not self._icon:
+            return QtGui.QIcon()
```

Comment:
Not sure if Qt designer calls this multiple times, but if so it'd be better to do this in the initializer (`self._icon = icon or QtGui.QIcon()`)

Reply:
Makes sense.

Reply:
Addressed via https://github.com/slaclab/pydm/pull/600/commits/19ede5c610dfc63dbd1d5f97001cc2503ff16390